src/styles/parts/nav.scss

```diff
@@ -0,0 +1,42 @@
+@import "../utils/_variables.scss";
+
+.nav {
+  padding: 0;
+  margin: 0;
```

Comment:
The same BEM issue - margin can't be on BEM block. .nav is BEM block. You have header above nav, you can use it as block (.header is BEM block, .nav <-- in this form it's also BEM block, where you can't style margins, even if it's physically possible and work on browser) and place on its child class .header__nav (<- it's BEM element where you can style margins ) and move all margins from .nav{} to .header__nav {} and your nav should be like this <nav class="nav header__nav>. And please check all your classes, I see margins set in many places of your BEM blocks also in other files.